l1-contracts/README.md

```diff
@@ -0,0 +1,304 @@
+# Nil L1-Contracts
+
+This repository contains all the L1 contracts that are to be deployed on mainnent & sepolia testnet
```

Comment:
could you please update readme. It' not a repository. See [example of internal readme style](https://github.com/NilFoundation/nil/blob/main/niljs/README.md)

Reply:
updated readme and restructured the directory as `rollup-bridge-contracts`